doc/Accessibility.md

```diff
@@ -0,0 +1,81 @@
+# Accessibility
+
+## Commands
```

Comment:
I think I did that, if there is any piece of information you think might be missing as far as initial support let me know, otherwise there is a lot of case by case depending on functional needs, for instance the situation with Commands in this PR, for those we will document as we implement the fixes.